network/core-node/admin-guide/logging.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Logging
+sidebar_position: 70
+---
+
+Stellar Core sends logs to standard error and `stellar-core.log` by default, configurable with the `LOG_FILE_PATH` field.
+
+Log messages are classified by progressive _priority levels_: `TRACE`, `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `FATAL`. The logging system only emits those messages at or above its configured logging level.
+
+Log messages at different priority levels can be color-coded on standard error by setting `LOG_COLOR=true` in the config file. By default they are not color-coded.
+
+The log level can be controlled by configuration, the `-ll` command-line flag, or adjusted dynamically by administrative (HTTP) commands. To do so, run:
+
+```bash
+$ stellar-core http-command "ll?level=debug"
+```
+
+while your node is running.
+
+Log levels can also be adjusted on a partition-by-partition basis through the administrative interface. For example the history system can be set to `DEBUG`-level logging by running:
+
```

Comment:
Since we are not listing out the different partitions here, how about having a link to [GH commands doc](https://github.com/stellar/stellar-core/blob/master/docs/software/commands.md#http-commands) where the full list is specifed (Bucket, Database, Fs, Herder, History, Ledger, Overlay, Process, SCP, Tx).

Reply:
Similar for log levels, FATAL, ERROR, WARNING, INFO, DEBUG, VERBOSE, TRACE

Reply:
Great suggestion! Adding a note for the partitions.

We do have the log levels specified higher-up in the page (although it was missing `VERBOSE` until this morning), though.